configs/sst_virtualization-firmware.yaml

```diff
@@ -0,0 +1,17 @@
+document: feedback-pipeline-workload
+version: 1
+data:
+  name: Virtualization Firmware
+  description: Virtual firmware packages like EDK2, seabios which support virtualization capabilities
+  maintainer: sst_virt_firmware
```

Comment:
The convention is that filenames are prefixed with the SST name.  Therefore, this file should be named `sst_virt_firmware-SOMETHING.yaml`.